dev-docs/bidders/smartx.md

```diff
@@ -0,0 +1,43 @@
+| `publisherId`        | required | A unique PublisherID.                                                                                                                                             | `'publisher_Id'`                                                                                                                                                                                      | `string`    |
+| `siteId`             | required | A unique SiteID.                                                                                                                                                  | `'site_id'`                                                                                                                                                                                           | `string`    |
+| `bidfloor`           | required | Value of Bidfloor.                                                                                                                                                |                              `0.3`                                                                                                                                                                                                 | `float`     |
+| `bidfloorcur`        | required | Used Currency. (e.g. EUR, USD etc.)                                                                                                                               | `'EUR'`                                                                                                                                                                                               | `string`    |
+| `context`            | required | Token that describes which context to play: 'instream' or 'outstream'                                                                                             | `'outstream'`                                                                                                                                                                                         | `string`    |
+| `outstream_function` | optional | Custom function to be used as a renderer.                                                                                                                         |                         `function(bid){console.log(bid);}`                                                                                                                                                                    | `function`  |
+| `outstream_options`  | optional | Object to set options on the smartx renderer.                                                                                                                     |                                `{}`                                                                                                                                                                                                  | `object`    |
```

Comment:
These are `required` for context `outstream`.

Reply:
Changed to required with further description